File: backTest/src/main/java/com/example/backTest/Entity/Generic/genericModel.java
package com.example.backTest.Entity.Generic;

import com.example.backTest.Entity.Base.baseModel;

import jakarta.persistence.Column;

public abstract class genericModel extends baseModel {
    @Column(name = "name", nullable = false, length = 100)
    private String name;

    @Column(name = "description", nullable = true, length = 255)
    private String description;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: CuidaTuAguaBackend/Entity/Model/genericModel/GenericModel.cs
using Entity.Model.baseModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entity.Model.genericModel
{
    public class GenericModel : BaseModel
    {
        public string Name { get; set; }  
        public string Description { get; set; }
        public string Url { get; set; }
    }
}
